tests/e2e/default/navigation/more-options-menu/offline-user.wdio-spec.js

```diff
@@ -0,0 +1,49 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const utils = require('../../../../utils');
+const seeder = require('./seeder');
+
+describe('Export enabled when there are items: messages, contacts, peope', async () => {
+  before(async () => {
+    await seeder.saveDocs();
+    await utils.createUsers([seeder.offlineUser]); //offline user
+    await seeder.newLogin(seeder.offlineUser);
+  });
+
+  it('- Message tab', async () => {
```

Comment:
The test case description should be clear and descriptive, for example: 

`should not display menu when no options are available`

Please apply to the other test cases below. Thanks!